socorro/unittest/external/es/test_crashstorage.py

```diff
@@ -225,7 +229,7 @@ def test_bulk_index_crash(self):
                 x,
                 a_processed_crash['uuid'][2:]
             )
-            # Ensure that the document was indexed by attempting to retreive it.
+            # Ensure that the document was indexed by attempting to retreive it
```

Comment:
Shouldn't there be a test that tries to retrieve it afterwards that checks that `memory_report` did NOT get stored.